ui/src/data-services/models/session.ts

```diff
@@ -71,6 +71,13 @@ export class Session {
   }
 
   get numSpecies(): number | undefined {
+    // Workaround for taxa count being 0 when occurrences count > 0
+    if (this.numOccurrences) {
+      if (this._event.taxa_count === 0) {
```

Comment:
In the future I think the backend should return null for taxa_count if the data has not been processed or taxa_count not yet populated (its a cached value).